common/export.py

```diff
@@ -442,3 +502,27 @@ def compute_brightway_impacts(activity, method, impacts_py):
         results[key] = float("{:.10g}".format(lca.score))
         logger.debug(f"{activity}  {key}: {lca.score}")
     return results
+
+
+def generate_compare_graphs(processes, impacts_py, graph_folder, output_dirname):
+    impacts_compared_dic = compare_impacts(
+        processes, settings.bw.ecoinvent, impacts_py, IMPACTS_JSON
+    )
```

Comment:
Nit: this function call would read a **whole lot better** with kwargs (same also applies for the caller fn)